Sources/Fuzzilli/FuzzIL/Analyzer.swift

```diff
@@ -169,7 +169,8 @@ struct ContextAnalyzer: Analyzer {
     mutating func analyze(_ instr: Instruction) {
```

Comment:
Given this bug, we should probably have tests for this. Unless you really feel like adding such a test, maybe just add a `// TODO add a test for this analyzer when modifying this function in the future` here?

Reply:
I've added a test for the Context Analyzer

Reply:
thanks!